gateway-js/src/__tests__/integration/sample-schema.ts

```diff
@@ -0,0 +1,391 @@
+
+export const sampleSchema = `
```

Comment:
This const is no longer referenced, we can delete this file

Reply:
got it. Thanks